.csslintrc

```diff
@@ -0,0 +1,12 @@
+{
+    "adjoining-classes": false,
+    "box-model": false,
+    "box-sizing": false,
+    "compatible-vendor-prefixes": false,
+    "duplicate-background-images": false,
+    "import": false,
+    "important": true,
+    "outline-none": false,
+    "overqualified-elements": false,
+    "text-indent": false
```

Comment:
The indentation looks wrong in all files! Though I had soft tabs turned off! fixing it now!!